api/src/data/migrator.ts

```diff
@@ -1,7 +1,7 @@
 import express, { Request, Response } from "express";
 import { join } from "path";
 
-import { db } from "./";
+import { db } from "./db-client";
```

Comment:
Prefer absolute paths for imports. i.e. `@/db/db-client` 

If the db client isn't at that location, then please standardize it to that location.
If this would cause a tonne of changes, then make an alias to the old location, with a JSDoc deprecation on the old location.